R/deployApp.R

```diff
@@ -247,7 +247,7 @@ deployApp <- function(appDir = getwd(),
   }
 
   # determine the deployment target and target account info
-  target <- deploymentTarget(appPath, appName, appTitle, appId, account, server)
+  target <- deploymentTarget(recordDir, appName, appTitle, appId, account, server)
```

Comment:
Internally, `deploymentTarget` calls this argument `appPath`; should it be renamed?

Reply:
I've renamed it to `recordPath`, in anticipation of renaming `recordDir` to `recordPath` in a future PR.